src/components/modals/ConfirmationModal.js

```diff
@@ -1,37 +1,64 @@
-import {button, div, h} from 'react-hyperscript-helpers';
-import {Styles} from '../../libs/theme';
+import React from 'react';
+import {div, h} from 'react-hyperscript-helpers';
 import Modal from 'react-modal';
 import CloseIconComponent from '../CloseIconComponent';
+import './ConfirmationModal.css';
+import {styled} from '@mui/material/styles';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
 
 const ConfirmationModal = (props) => {
   const {showConfirmation, closeConfirmation, title, message, header, onConfirm, styleOverride = {}} = props;
   const closeFn = () => closeConfirmation();
 
-  const confirmButton = button({className: 'cell-button hover-color', onClick: onConfirm}, ['Confirm']);
-  const computedStyle = Object.assign({}, Styles.MODAL.CONFIRMATION, styleOverride);
+  const duosBlue = 'rgb(0, 96, 159)';
+  const duosBlueHover = 'rgb(9,72,183)';
+
+  const SecondaryButton = styled(Button)(() => ({
```

Comment:
I think these styles would be great candidates for the global styles declaration, however we choose to handle those in the future! :)

Reply:
Agreed - I was pulling these out explicitly for that reason.